### DISCUSSION TITLE
Bulk editing of due date for sub tasks

### DISCUSSION THREAD

**Discussion by syreali** [kanboard/kanboard#5431] [General]
Hi,
I have around 15 subtasks within a task. I want to set a due date for each of the sub task as (for example) 10 March 2024. I can't figure out how to do that in one step.
Currently I have to open each sub task separately to set the due date, which means doing this 15 times, which is very inefficient.

Could you please guide how to improve my process.
Thanks
Rehan

**Comment by fguillot** [comment]
By default, Kanboard does not have due date for subtasks. You must be using a third-party plugin. I recommend reaching out to the plugin author.
